src/pages/login/login.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { GoogleLogin } from "react-google-login";
-import { Button } from "../../components/Button/Button";
+import { Button } from "@/components/Button/Button";
```

Comment:
I'm afraid that eslint does not understand your aliases - see results of the lint job - https://github.com/akai-org/EaterUI/runs/4401012684?check_suite_focus=true

I managed to fix it in 890733429acbba3094ee4f40b16e1630a7429da1 - let me know if you're ok with that - otherwise feel free to drop the commit 😉

Reply:
It's fine, awaiting for merge 😀